ORBITAL_INTEGRATION.s

```diff
@@ -227,7 +227,7 @@ GAMCOMP		VLOAD	VSR1
 # Page 1339
 			36D
 		STORE	BETAM
-		NORM	BDDV		# FORM NORMALIZE QUOTIEN ALPHAM/BETAM
+		NORM	BDDV		# FORM NORMALIZE QUOTIENT ALPHAM/BETAM
 			33D
 		SR1R	PUSH		# C(PDL+2) = ALMOST NORMALIZE RHO.
```

Comment:
Non-spotted typo. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1339.jpg)

`NORMALIZE` should be `NORMALIZED`